src/Components/Blazor/Build/src/targets/StaticWebAssets.targets

```diff
@@ -0,0 +1,30 @@
+<Project>
+
+  <PropertyGroup>
+    <ResolveStaticWebAssetsInputsDependsOn>
+      $(ResolveStaticWebAssetsInputsDependsOn);
+      _RemoveBlazorWwwrootFromStaticWebAssets;
+    </ResolveStaticWebAssetsInputsDependsOn>
+  </PropertyGroup>
+
+
+  <Target Name="_RemoveBlazorWwwrootFromStaticWebAssets">
+    <ItemGroup>
+      <StaticWebAsset Remove="@(StaticWebAsset)" Condition="'%(SourceType)' == ''" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="BlazorStaticWebAssetsComputeFilesToPublish"
+    AfterTargets="_StaticWebAssetsComputeFilesToPublish">
+  
+    <ItemGroup>
+      <_StandaloneExternalPublishStaticWebAsset Include="@(_ExternalPublishStaticWebAsset)" Condition="'%(RelativePath)' != ''">
+        <RelativePath>$([MSBuild]::MakeRelative('$(MSBuildProjectDirectory)', '$([MSBuild]::NormalizePath('$([System.String]::Copy('%(RelativePath)').Replace('wwwroot','$(BlazorPublishDistDir)'))'))'))</RelativePath>
+      </_StandaloneExternalPublishStaticWebAsset>  
+
+      <ResolvedFileToPublish Remove="@(_StandaloneExternalPublishStaticWebAsset)" />
+      <ResolvedFileToPublish Include="@(_StandaloneExternalPublishStaticWebAsset)" />
```

Comment:
This seems odd. Why do we both remove and then re-add the same list of things? Is it to refresh some metadata? Could you add a comment to clarify what's going on here?

Reply:
Update doesn't work. I already tried this.

Reply:
Is this a bug in msbuild? Can you add a comment to clarify whether it's a known bug? Otherwise we'll keep wondering why the code does this.

Reply:
It's kinda well known that `Update` doesn't work inside targets: https://github.com/microsoft/msbuild/issues/2835

Reply:
OK, wasn't well known to me. Maybe add a comment pointing to that URL for reference. Then it will become better known :)

Reply:
I've added a comment.